internal/docs/jacc/JACCSpecAssertions.xml

```diff
@@ -1468,28 +1448,19 @@ permission being checked. If checkPermission throws an
 AccessControlException, the checked permission is not granted. Otherwise the
 permission is granted.
 
-- The J2EE 1.4 container calls Policy.implies with two arguments; the
+- The container calls Policy.implies with two arguments; the
 permission being checked and a ProtectionDomain that need not be
 constructed with principals. The checked permission is granted if
 Policy.implies returns true. Otherwise, the permission is not granted.
 
-- The J2EE 1.4 container calls
+- The container calls
```

Comment:
+1 as this implies the current Jakarta EE version that the TCK documentation is part of.